cf-serverd/server_common.c

```diff
@@ -376,9 +376,16 @@ static void AbortTransfer(ConnectionInfo *connection, char *filename)
     Log(LOG_LEVEL_VERBOSE, "Aborting transfer of file due to source changes");
 
     char sendbuffer[CF_BUFSIZE];
-    snprintf(sendbuffer, CF_BUFSIZE, "%s%s: %s",
+    int ret = snprintf(sendbuffer, CF_BUFSIZE, "%s%s: %s",
              CF_CHANGEDSTR1, CF_CHANGEDSTR2, filename);
 
+    if (ret < 0 || (size_t) ret >= CF_BUFSIZE)
+    {
+        Log(LOG_LEVEL_WARNING,
+            "Error message truncated in abort transfer transaction: "
+            "Filename too long");
```

Comment:
I am not sure why we print this warning but then don't fail/error-out? I guess this is a void function and the operation is to abort the transfer so that makes sense.

Reply:
Ok. Now I see some more comments above and context. Good.